# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ladspa-vcf/compiler.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- vcf-0.0.5.orig/Makefile	2002-08-12 14:53:36.000000000 +0200
+++ vcf-0.0.5/Makefile	2008-02-10 15:37:18.000000000 +0100
@@ -1,9 +1,9 @@
 all: vcf_cv_in.so vcf.so
 vcf_cv_in.so: vcf_cv_in.o 
-	gcc -shared -o vcf_cv_in.so vcf_cv_in.o -lm
+	$(CC) -shared -o vcf_cv_in.so vcf_cv_in.o -lm
 vcf.so: vcf.o 
-	gcc -shared -o vcf.so vcf.o -lm
+	$(CC) -shared -o vcf.so vcf.o -lm
 vcf_cv_in.o: vcf.so.c                
-	cc -c -fPIC -o vcf_cv_in.o vcf.so.c -DWITH_CV_IN
+	$(CC) -c -fPIC -o vcf_cv_in.o vcf.so.c -DWITH_CV_IN
 vcf.o: vcf.so.c                
-	cc -c -fPIC -o vcf.o vcf.so.c
+	$(CC) -c -fPIC -o vcf.o vcf.so.c
